tests/integration/lsp_tests.rs

```diff
@@ -11917,6 +11917,11 @@ fn lsp_node_modules_dir() {
 
   assert!(!temp_dir.path().join("node_modules").exists());
 
+  // a lockfile will be created here because someone did an explicit cache
+  let lockfile_path = temp_dir.path().join("deno.lock");
+  assert!(lockfile_path.exists());
```

Comment:
Running deno cache creates a lockfile here so I think this is ok?

Reply:
Yes, I agree